Commands/CLI/help.js

```diff
@@ -0,0 +1,13 @@
+const { cli: commands } = require("../../Configurations/commands");
+
+module.exports = async () => {
+	const str = Object.keys(commands).map(c => {
+		const cmd = commands[c];
+		return `${cmd.isMultiline ? "*" : ""}${c}${cmd.usage ? ` ${cmd.usage}` : ""}${cmd.description ? ` - ${cmd.description}` : ""}`;
+	});
+	winston.info(" ======================== COMMAND LIST ========================");
+	str.forEach(s => winston.info(s));
```

Comment:
str.forEach(winston.info); :P

Reply:
No. [Array.forEach()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach) also calls the function passed with the item index and the array itself, which isn't a good idea.

Reply:
Oh right. Disregard.